dot/rpc/modules/author.go

```diff
@@ -35,8 +36,17 @@ type AuthorModule struct {
 	txStateAPI TransactionStateAPI
 }
 
+// HasSessionKeyRequest is used to receive the rpc data
+type HasSessionKeyRequest struct {
+	Data string
+}
```

Comment:
what is the data? a public key or a list of public keys?

Reply:
This is a string of concatenated public keys in hex. Changed to `PublicKeys`